headers/functions/overlay29.h

```diff
@@ -833,6 +845,11 @@ void InitPortraitDungeon(struct portrait_params* portrait, enum monster_id monst
                          enum portrait_emotion emotion);
 void OpenMessageLog(undefined4 param_1, undefined4 param_2);
 bool RunDungeonMode(undefined4* param_1, undefined4 param_2);
+void StartFadeDungeon(struct dungeon_fade* fstruct, int delta_delta_brightness,
+                      enum fade_status_dungeon fade_type);
+void StartFadeDungeonWrapper(int fade_type, int delta_delta_brightness, enum screen screen);
```

Comment:
Is `delta_delta` a typo?

```suggestion
void StartFadeDungeon(struct dungeon_fade* fstruct, int delta_brightness,
                      enum fade_status_dungeon fade_type);
void StartFadeDungeonWrapper(int fade_type, int delta_brightness, enum screen screen);
```

Reply:
No, it's not a typo. I initially found the general (non dungeon) fade function, where I named the field that changes the screen brightness as `delta_brightness`. This is because it gets positive/negative values, so it's essentially representing a change in screen brightness.

Now it's also here in the dungeon mode equivalent, but instead of tracking frames remaining in the fade and calculating the delta_brightness from there, here it seems to just specify a _change_ in the delta value per call. Hence the `delta_delta`.

It's a pretty confusing name, but I couldn't come up with a better one, so I'm taking suggestions lol.

Reply:
I see. I can't think of anything better, so whatever 🤷